packages/components/order-tracking/src/index.scss

```diff
@@ -0,0 +1,11 @@
+.order-tracking {
+  @import "~bootstrap4/scss/_functions.scss";
```

Comment:
Is everything actually used ?

Reply:
yes, some of the bootstrap imports are required by others